LearningMissionSimulation/PlaygroundMher.cs

```diff
@@ -4,59 +4,61 @@
 
 namespace LearningMissionSimulation
 {
-    public class PlaygroundMher
+    public class PlaygroundMher : ISimulation
     {
-        public PlaygroundMher()
-        {
-            
-        }
-       
-        //Account account = ObjectGenerator.GenerateAccount();
+        List<Account> studentList = new List<Account>();
+
+        List<Account> instructorList = new List<Account>();
 
         List<Account> accountList = new List<Account>();
```

Comment:
Why do you need this? You created accountDictionary to store all accounts, and you created accountQueue to manage account activation. Please remove it.